docs/source/devel/repository_service_tuf.cli.admin.rst

```diff
@@ -12,6 +12,14 @@ repository\_service\_tuf.cli.admin.ceremony module
    :undoc-members:
    :show-inheritance:
 
+repository\_service\_tuf.cli.admin.import\_targets module
+---------------------------------------------------------
+
+.. automodule:: repository_service_tuf.cli.admin.import_targets
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
Is this an unrelated change?

Reply:
It is related to the doc build.
It wasn't automatically updating the implementation docs.